sdk/examples/intkey_javascript/index.js

```diff
@@ -17,7 +17,7 @@
 
 'use strict'
 
-const {TransactionProcessor} = require('sawtooth-sdk/processor')
```

Comment:
Were these changes based on the output from Standard.js?  If not, this commit should be dropped

Reply:
`standard` is not very rigorous. In particular, it doesn't care about line length (see https://github.com/standard/standard/issues/28). But the long lines certainly need to be taken care of, if for no other reason than that they make the docs look bad:

<img width="1137" alt="screen shot 2017-12-18 at 2 04 32 pm" src="https://user-images.githubusercontent.com/25748894/34126657-f590ce60-e3ff-11e7-963a-396015cc032c.png">

I don't care about stuff like 
```
-const {Message} = require('../protobuf')
+const { Message } = require('../protobuf')
```
but I do think that most of these changes are worth making.

Reply:
I just want to ensure that the changes being made are consistent with Standard JS.  If so, I'll approve it.

Reply:
You could also pull the Javascript changes into a separate PR, since they're unrelated to fixes  to get the build to pass

Reply:
The changes here are a strict superset of what is required by `standard`.